.github/workflows/auto_add_pr_to_miletone.yml

```diff
@@ -10,17 +10,17 @@ on:
 jobs:
   add_to_milestone:
     if: github.event.pull_request.merged == true && startsWith(github.event.pull_request.title, '[Version Bump]') == false
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-24.02
```

Comment:
Shouldn't it be `22.04` instead? (same below)

Reply:
Good catch, it should be `24.04` instead of `02`.
As of now, `ubuntu-latest` is 22.04, but starting December 5 it will be 24.04, so this will upgrade an pin.